src/components/metadata/metadata.test.js

```diff
@@ -227,53 +228,66 @@ describe('MetaData', () => {
   });
 
   describe('Parameter nodes', () => {
-    it('shows the node type as an icon', () => {
-      const wrapper = mount({ nodeId: rabbitParamsNodeId });
-      expect(
-        rowIcon(wrapper).hasClass('pipeline-node-icon--icon-parameters')
-      ).toBe(true);
-    });
-
-    it('shows the node name as the title', () => {
-      const wrapper = mount({ nodeId: rabbitParamsNodeId });
-      expect(textOf(title(wrapper))).toEqual(['Params:rabbit']);
-    });
-
-    it('shows the node type as text', () => {
-      const wrapper = mount({ nodeId: rabbitParamsNodeId });
-      const row = rowByLabel(wrapper, 'Type:');
-      expect(textOf(rowValue(row))).toEqual(['parameters']);
-    });
-
-    it('shows the node filepath', () => {
-      const wrapper = mount({ nodeId: rabbitParamsNodeId });
-      const row = rowByLabel(wrapper, 'File Path:');
-      expect(textOf(rowValue(row))).toEqual(['-']);
-    });
-
-    it('shows the node parameters', () => {
-      const wrapper = mount({ nodeId: rabbitParamsNodeId });
-      const row = rowByLabel(wrapper, 'Parameters (-):');
-      expect(textOf(rowValue(row))).toEqual(['-']);
-    });
-
-    it('shows the node tags', () => {
-      const wrapper = mount({ nodeId: rabbitParamsNodeId });
-      const row = rowByLabel(wrapper, 'Tags:');
-      expect(textOf(rowValue(row))).toEqual(['Small']);
-    });
-
-    it('shows the node pipeline', () => {
-      const wrapper = mount({ nodeId: rabbitParamsNodeId });
-      const row = rowByLabel(wrapper, 'Pipeline:');
-      expect(textOf(rowValue(row))).toEqual(['Default']);
+    describe('when parameters are enabled', () => {
```

Comment:
I thought these changes were already merged in the parameter hiding branch? Do you need to sync this with main maybe?